CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
 # Changelog
 
 All notable changes to `SyliusTpayPlugin` will be documented in this file.
+
+## 3.0
```

Comment:
needs real update - I bet it is not all of it (I didn't mention anything about API Platform)